feature-detects/proxy.js

```diff
@@ -0,0 +1,21 @@
+/*!
+{
+  "name": "Proxy Object",
+  "property": "Proxy",
+  "caniuse": "proxy",
```

Comment:
you can add a authors tag here with you name/userhandle if you want :-)

Reply:
Added, thanks for the suggestion